src/retroshare/rsconfig.h

```diff
@@ -389,17 +389,30 @@ class RsServerConfig
 	 * @param[in] opModeStr new operating mode as string
 	 * @return
 	 */
-    virtual bool     setOperatingMode(const std::string &opModeStr) = 0;
+	virtual bool     setOperatingMode(const std::string &opModeStr) = 0;
 
-    /* Data Rate Control - to be moved here */
+	/* Data Rate Control */
 	/**
 	 * @brief SetMaxDataRates set maximum upload and download rates
 	 * @jsonapi{development}
-	 * @param[in] downKb download rate in kB
-	 * @param[in] upKb upload rate in kB
+	 * @param[in] inKb download rate in kB
+	 * @param[in] outKb upload rate in kB
 	 * @return returns 1 on succes and 0 otherwise
 	 */
-    virtual int SetMaxDataRates( int downKb, int upKb ) = 0;
+	RS_DEPRECATED_FOR("setMaxDataRates( int inKb, int outKb, int inKbWhenIdle, int outKbWhenIdle)") \
+	virtual int SetMaxDataRates( int inKb, int outKb) = 0;
```

Comment:
Param names must remain the same too, otherwise JSON API retro-compatibility will be broken